quickstart/monitoring/monitoring-quickstart.ipynb

```diff
@@ -71,16 +71,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "TRUERA_URL = \"https://app.truera.net/\"\n",
-    "AUTH_TOKEN = \"<replace with your token>\""
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "32570cab",
-   "metadata": {},
-   "source": [
-    "### Step 2: Install packages"
+    "TRUERA_URL = \"https://app.truera.net\"\n",
```

Comment:
Are we using `app-demo.truera.net` or `app.truera.net`

Reply:
The one we share with the client will have app-demo this is pointing to app.truera.net as this is public.